lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -17,10 +17,14 @@ class GithubRelease # rubocop:disable Metrics/ClassLength
     def_delegator :@build_mechanism, :output_file
 
     # @param build_mechanism Delegates building after GitHub release is created.
-    def initialize(build_mechanism, ci_status_timeout: 600, max_tag_find_timeout: 240)
+    def initialize(build_mechanism,
+                   ci_status_timeout: 600,
+                   max_tag_find_timeout: 240,
+                   skip_ci_status: false)
       @build_mechanism = build_mechanism
       @ci_status_timeout = ci_status_timeout
       @max_tag_find_timeout = max_tag_find_timeout
+      @skip_ci_status = skip_ci_status
```

Comment:
Instead of supporting both this and the moonshot config, could we set this to the value passed (if one is) or default to the moonshot config value? `@skip_ci_status = skip_ci_status.nil? ? Moonshot.config.skip_ci_status : skip_ci_status` or something to that effect.

Reply:
Resolved, this is no longer required.